python/erdos/streams.py

```diff
@@ -140,6 +146,8 @@ def split_fn(serialized_data: bytes) -> bool:
         left_stream, right_stream = self._internal_stream._split(split_fn)
         return (OperatorStream(left_stream), OperatorStream(right_stream))
 
+#"""LOOK UP MAYBE"""
+
```

Comment:
Please remove.

Reply:
The `split_by_type` function is a tricky one to provide types for. IMO I would go for the following:

```python
def split_by_type(self, *data_type: Type) -> Tuple["OperatorStream[Any]", ...]:
```

Reply:
Not sure about my solution , but gave it a try.